reana_client/validation.py

```diff
@@ -431,9 +594,9 @@ def parse_params(stages):
     command_params = set(cmd_param_steps_mapping.keys())
 
     # REANA input parameters validation
-    _warn_not_used_parameters(input_parameters, workflow_params)
+    rest_workflow_params = _warn_not_used_parameters(input_params, workflow_params)
     # Yadage parameters validation
-    _warn_not_used_parameters(workflow_params, command_params, type_="Yadage")
+    _warn_not_used_parameters(rest_workflow_params, command_params, type_="Yadage")
```

Comment:
```suggestion
    _warn_not_used_parameters(rest_workflow_params, command_params, type_="yadage")
```

Reply:
This is used to print the warning message. If changed the result would be "WARNING: yadage input parameter "input_histfitter" does not seem to be used.", whereas in other warnings we have "Yadage".